src/libraries/sendtohelixhelp.proj

```diff
@@ -37,8 +37,9 @@
         '$(Scenario)' == 'gcstress0xc_jitstress1' or
         '$(Scenario)' == 'gcstress0xc_jitstress2' or
         '$(Scenario)' == 'gcstress0xc_jitminopts_heapverify1'">06:00:00</_workItemTimeout>
-    <_workItemTimeout Condition="'$(_workItemTimeout)' == '' and ('$(TargetOS)' == 'iossimulator' or '$(TargetOS)' == 'tvossimulator' or '$(TargetOS)' == 'maccatalyst' or '$(TargetOS)' == 'android')">00:30:00</_workItemTimeout>
-    <_workItemTimeout Condition="'$(_workItemTimeout)' == '' and ('$(TargetOS)' == 'ios' or '$(TargetOS)' == 'tvos')">00:45:00</_workItemTimeout>
+    <_workItemTimeout Condition="'$(_workItemTimeout)' == '' and ('$(TargetOS)' == 'iossimulator' or '$(TargetOS)' == 'tvossimulator' or '$(TargetOS)' == 'maccatalyst')">03:00:00</_workItemTimeout>
```

Comment:
I don't know how I feel about this change. This has the potential to back things up considerably should the work item go on and on.

Reply:
It seems that simulators are too slow. Since there is a coverage on devices, we may reduce the testing scope on the simulators, disabling the `JIT` and `SIMD` subsets for example.

Reply:
3 hours seems like too much to me. Is it truly that slow?

Reply:
Where does it spend the most amount of time: building vs execution ?

Reply:
<img width="328" alt="Screenshot 2023-09-20 at 18 40 03" src="https://github.com/dotnet/runtime/assets/55002338/4b440042-4248-4c10-866f-b949508447b7">

Reply:
Execution is time consuming on simulators. Building tests takes about 40 mins on both devices and simulators. Execution takes about 40 mins on devices and about 2h on simulators. On tvossimulator the execution timed out after almost 3h.

Reply:
@rolfbjarne do you also experience long execution times on ios-* simulators on xamarin's CI?

Reply:
> Execution is time consuming on simulators. Building tests takes about 40 mins on both devices and simulators. Execution takes about 40 mins on devices and about 2h on simulators. On tvossimulator the execution timed out after almost 3h.

Are we building runtime tests on helix?

Reply:
The tests are first built on the build machine and then AOT-compiled on Helix.

Reply:
> @rolfbjarne do you also experience long execution times on ios-* simulators on xamarin's CI?

Usually just a few minutes, and it's not changed recently. Then again, we don't have that many tests either.

I find it weird that the simulator is so much slower than device though for you (assuming you run the same set of tests for both), in our experience simulator has always been the faster of the two.

Reply:
I think the test execution on simulators is not identical to the devices. For example, for `tracing/eventpipe/buffersize` the test duration is almost the same. However, there is a log on simulators after each test that kills the simulator, which may stale the execution:
```
info: Application has finished with exit code: 100 (as expected)
info: Cleaning up simulator 'iPhone X (iOS 15.0) - created by XHarness'
dbug: 
dbug: Running launchctl remove com.apple.CoreSimulator.CoreSimulatorService
dbug: Process simctl exited with 137
dbug: Process launchctl exited with 0
dbug: 
dbug: Running killall -9 "iPhone Simulator" "iOS Simulator" Simulator "Simulator (Watch)" com.apple.CoreSimulator.CoreSimulatorService ibtoold
dbug: Process killall exited with 0
info: Simulators cleaned up
dbug: Saving diagnostics data to '/tmp/helix/working/BA6B0A0B/w/9E9B08FA/e/diagnostics.json'
XHarness exit code: 0
```

Reply:
The simulator runs get unstable after running for a while. The shutdown / restart is meant to protect against that.

Reply:
It seems that the `xcrun simctl shutdown` and `xcrun simctl boot` are time-consuming. Without the `--reset-simulator` parameter, the tests take the same amount of time as they do for devices.

I suggest disabling it for the runtime tests and monitoring the CI.